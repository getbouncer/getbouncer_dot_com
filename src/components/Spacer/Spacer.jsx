import React from 'react'
import PropTypes from 'prop-types'
import Height from './styles'

const Spacer = ({ height }) => <Height height={height} />

Spacer.propTypes = {
  height: PropTypes.number.isRequired,
}

export default Spacer
