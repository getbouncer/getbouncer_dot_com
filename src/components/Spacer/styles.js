import styled from '@emotion/styled'

export default styled.div`
  height: ${props => props.height}px;
`
