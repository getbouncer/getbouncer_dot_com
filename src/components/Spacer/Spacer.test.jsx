import React from 'react'
import renderer from 'react-test-renderer'
import { describe, expect, it } from 'jest'
import Spacer from './Spacer'

describe('Spacer', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Spacer height={20} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
