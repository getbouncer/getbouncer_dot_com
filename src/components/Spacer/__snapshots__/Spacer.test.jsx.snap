// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer renders correctly 1`] = `
<div
  className="css-1j0bswb"
  height={20}
/>
`;
